app/lib/l10n/de.arb

```diff
@@ -175,7 +180,7 @@
       }
     }
   },
-  "moveFailure": "Leider konnte der Patient nicht erfolgreich bewegt werden.\nBitte versuche es erneut.",
+  "moveFailure": "Leider konnte der Patient nicht erfolgreich bewegt werden. Bitte versuche es erneut.",
 
 
   "resourceDirectorySelectLocation": "Auswählen",
```

Comment:
I can't comment on the unchanged lines below, but we could also change the selectScreen... entries to homeScreen...